test/units/testsuite-17.credentials.sh

```diff
@@ -4,6 +4,11 @@
 set -eux
 set -o pipefail
 
+if ! systemctl is-enabled systemd-udev-load-credentials.service; then
```

Comment:
This is too _drastic_, as the unit doesn't seem to be enabled by default (at least not by our presets), so the test is now being skipped in CIs:

```
[   67.653934] testsuite-17.sh[382]: + .//usr/lib/systemd/tests/testdata/units/testsuite-17.credentials.sh
[   67.661248] testsuite-17.sh[3742]: + set -o pipefail
[   67.661248] testsuite-17.sh[3742]: + systemctl is-enabled systemd-udev-load-credentials.service
[   67.675227] testsuite-17.sh[3743]: disabled
[   67.675709] testsuite-17.sh[3742]: + echo no systemd-udev-load-credentials.service
[   67.675832] testsuite-17.sh[3742]: + exit 0
```

Changing this to something like
```suggestion
if [[ $(systemctl is-enabled systemd-udev-load-credentials.service) == not-found ]]; then
```
should help.